modular-docs-manual/content/topics/creating_procedure_modules.adoc

```diff
@@ -18,29 +27,37 @@ The introduction typically provides context for the procedure, such as:
 * Special considerations specific to the procedure
 
-.Procedure prerequisites
-This section is optional. Prerequisites are a bulleted list of conditions that must be satisfied before the user starts the procedure. Use a bulleted list for prerequisites and the plural heading *Prerequisites*, even if you only have a single prerequisite.
+.Limitations
+This section is optional and not used often. If there are any limitations that the affect the procedure, list them in this section.
+
+.Prerequisites
+This section is optional. Prerequisites are a bulleted list of conditions that must be satisfied before the user starts the procedure. Use a bulleted list for prerequisites and the plural heading *Prerequisites*, even if your procedure only has a single prerequisite.
 
 Prerequisites can be full sentences or sentence fragments; however, prerequisite list items must be parallel.
 
 Focus on relevant prerequisites that users might not otherwise be aware of. Do not list obvious prerequisites. If a prerequisite applies to all of the procedures in a user story, consider listing the prerequisite in the assembly file. If you do this, consider including the prerequisite in the procedure module as a comment.
 
-.Procedure steps
-The procedure consists of one or more steps required to complete the procedure. Each step describes one action. For single-step procedures, use an unnumbered bullet instead of a numbered list.
+.Procedure
+This section is required. The procedure consists of one or more steps required to complete the procedure. Each step describes one action written in the imperative form, for example, 'Do this action'. For single-step procedures, use an unnumbered bullet instead of a numbered list.
 
 NOTE: Not all numbered lists in documentation are procedures. You can also use numbered lists in any module type for non-procedural sequences, such as a process flow of system actions.
 
-.Procedure verification
+.Verification or Verification steps
 This section is optional. It provides the user with one or more steps to verify that the procedure provided the intended outcome, for example:
 
 * An example of expected command output or a pop-up window that the user receives when the procedure is successful
 * Actions for the user to complete, such as entering a command, to determine the success or failure of the procedure
 
-.Procedure additional resources
-This section is optional. The additional resources list links to other material closely related to the contents of the procedure module, such as other documentation resources, instructional videos, or labs.
+.Troubleshooting
+This section is optional and not used often. List any actions that can help with troubleshooting the procedure. This subsection should be short. Consider whether the information might be better in a separate troubleshooting procedure or as part of a reference module that contains other troubleshooting sections.
+
+.Next steps and Additional resources
```

Comment:
If it's optional to use both Next steps and Additional resources, would it make sense to separate out the explanation for each section, like the other options? Is the order implied then by the order of the list and explanations, and could the line about putting Next steps first be removed here?

Reply:
I agree with this idea, it would make it easier then for a writer to implement both of these sections, if they needed to, if they were separately described.